core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -3210,11 +3213,292 @@ void checkCorrelatedMapSubQuery(boolean expand) {
     sql(sql2).ok();
   }
 
+
+  @Disabled("JOIN on (X IN Y) fails in Calcite (TODO: find existing issue if it exists)")
+  @Test void testJoinIn() {
```

Comment:
Why is this broken???? Is this unrelated?

Reply:
Oh also this is a new test.

Reply:
This is the unrelated issue that I mentioned I was punting on.

Reply:
Sounds good.